workers/loc.api/sync/dao/db-migrations/sqlite-migrations/migration.v19.js

```diff
@@ -0,0 +1,353 @@
+      `CREATE INDEX fundingOfferHistory_user_id_status_mtsUpdate
+        ON fundingOfferHistory(user_id, status, mtsUpdate)`,
+      `CREATE INDEX fundingOfferHistory_user_id_mtsUpdate
+        ON fundingOfferHistory(user_id, mtsUpdate)`,
+      `CREATE INDEX fundingOfferHistory_user_id_subUserId_mtsUpdate
+        ON fundingOfferHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX fundingLoanHistory_user_id_symbol_mtsUpdate
+        ON fundingLoanHistory(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX fundingLoanHistory_user_id_amount_mtsUpdate
+        ON fundingLoanHistory(user_id, amount, mtsUpdate)`,
+      `CREATE INDEX fundingLoanHistory_user_id_status_mtsUpdate
+        ON fundingLoanHistory(user_id, status, mtsUpdate)`,
+      `CREATE INDEX fundingLoanHistory_user_id_mtsUpdate
+        ON fundingLoanHistory(user_id, mtsUpdate)`,
+      `CREATE INDEX fundingLoanHistory_user_id_subUserId_mtsUpdate
+        ON fundingLoanHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX fundingCreditHistory_user_id_symbol_mtsUpdate
+        ON fundingCreditHistory(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX fundingCreditHistory_user_id_amount_mtsUpdate
+        ON fundingCreditHistory(user_id, amount, mtsUpdate)`,
+      `CREATE INDEX fundingCreditHistory_user_id_status_mtsUpdate
+        ON fundingCreditHistory(user_id, status, mtsUpdate)`,
+      `CREATE INDEX fundingCreditHistory_user_id_mtsUpdate
+        ON fundingCreditHistory(user_id, mtsUpdate)`,
+      `CREATE INDEX fundingCreditHistory_user_id_subUserId_mtsUpdate
+        ON fundingCreditHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX positionsHistory_user_id_symbol_mtsUpdate
+        ON positionsHistory(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_basePrice_mtsUpdate
+        ON positionsHistory(user_id, basePrice, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_mtsUpdate_mtsCreate
+        ON positionsHistory(user_id, mtsUpdate, mtsCreate)`,
+      `CREATE INDEX positionsHistory_user_id_mtsUpdate
+        ON positionsHistory(user_id, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_subUserId_mtsUpdate
+        ON positionsHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+      `CREATE INDEX positionsHistory_subUserId_id
+        ON positionsHistory(subUserId, id)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX positionsSnapshot_user_id_mtsUpdate
+        ON positionsSnapshot(user_id, mtsUpdate)`,
+      `CREATE INDEX positionsSnapshot_user_id_symbol_mtsUpdate
+        ON positionsSnapshot(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX positionsSnapshot_user_id_subUserId_mtsUpdate
+        ON positionsSnapshot(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX logins_user_id_ip_time
```

Comment:
Woudnt add indexes to logins with exception of last two:

`CREATE INDEX logins_user_id_time
        ON logins(user_id, time)`,
      `CREATE INDEX logins_user_id_subUserId_time
        ON logins(user_id, subUserId, time)
        WHERE subUserId IS NOT NULL`,

Reply:
Maybe we should take care of UI filter in this case and below

Reply:
yes, does not matter index for the filters.